tests/MoBi.Tests/Core/Commands/RenameFromContextMenuCommandForBuildingBlockSpecs.cs

```diff
@@ -0,0 +1,60 @@
+using System.Collections.Generic;
+using FakeItEasy;
+using MoBi.Core.Events;
+using MoBi.Presentation;
+using MoBi.Presentation.Presenter;
+using MoBi.Presentation.Tasks.Edit;
+using MoBi.Presentation.Tasks.Interaction;
+using MoBi.Presentation.UICommand;
+using OSPSuite.BDDHelper;
+using OSPSuite.Core.Domain;
+using OSPSuite.Core.Domain.Builder;
+
+namespace MoBi.Core.Commands
+{
+   public abstract class RenameFromContextMenuCommandForBuildingBlockSpecs : ContextSpecification<RenameFromContextMenuCommand<MoleculeBuildingBlock>>
+   {
+      protected IMoBiApplicationController _applicationController;
+      protected IInteractionTaskContext _interactionTaskContext;
+      private EditTasksForBuildingBlock<MoleculeBuildingBlock> _editTasks;
+      private ISelectRenamingPresenter _selectRenamingPresenter;
```

Comment:
I think this is unused and can be removed. Can you try? You'll have to remove some of the fakes associated with it, but they don't have any behaviors on them.

Reply:
Also, check _applicationController.